phac-nml/irida-next
# Samples table enhancements followup

## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR is a follow up to #679.

* Added in action cable config and updated the Projects Table so that it is now refreshed when a Sample is created, updated, or deleted.
* Moved hidden input handling from `infinite-scroll` controller into a new `form--hidden-inputs` controller.
* Updated `clone` and `transfer` samples response to reload the page
* Added `data-turbo-permanent` to `dialog` and `flash` so that if the page is refreshed they persist.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Launch server
2. Navigate to Project Samples page for Project of choice
3. Perform actions on the table that require a selection of samples and ensure that each respond as expected

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
